doc/src/explanation/gotchas.rst

```diff
@@ -806,4 +806,4 @@ These will give you the function parameters and docstring for
 
 .. module:: sympy.simplify.simplify
 .. autofunction:: powsimp
-   :noindex:
+   :no-index:
```

Comment:
Is this possible it will fail on Sphinx < 7.2.0?

Reply:
Yes, I have just confirmed that locally.

Reply:
I've removed that commit